tests/MyGame/Example/Color.d

```diff
@@ -0,0 +1,11 @@
+// automatically generated, do not modify
+
+module MyGame.Example.Color;
+
+enum Color : byte
+{
+   Red = 1,
```

Comment:
Enum members should be lower case according to the [D style guide](http://dlang.org/dstyle.html).